documentation/smg/15_documentation.md

```diff
@@ -88,7 +82,43 @@ More info can be found https://www.sphinx-doc.org/en/master/usage/restructuredte
 
 The contents list is a manual table of contents linking to main documentation which will appear on the main page. Any contributions which wish to be listed in the main table of contents should appear here.
 
+## Updating documentation
+
+When producing a new version, the `conf.py` (in `documentation/user_docs/docs`) should have the version updated to match that of the version being built. The documentation should be built manually (as described [here](#building-docs))
+
 ## Deploying documentation
 
-The documentation will be deployed automatically on a nightly build to the `latest` page of the GitHub pages. 
-On release, documentation will be deployed to the folder corresponding to the version number of the relase and redirected to by the `stable` page of the GitHub pages
+### Automatic procedure (recommended)
+The documentation will be deployed automatically on a nightly build to the `unstable` page of the GitHub pages (`https://pace-neutrons.github.io/Horace/unstable`). 
+On release, documentation will be deployed to the folder corresponding to the version number of the relase and redirected to by the `stable` page of the GitHub pages.
+
+### Manual procedure
+This should be used only in dire circumstances i.e. if forcing an update to old documentation or Jenkins failure, and not as the main method. 
+
+The raw docs are stored on the `gh-pages` branch of the main Horace repository. 
+
+First the project should be checked out to the version whose docs need to be updated. 
+
+The docs should be built (see [here](#building-docs)) and the appropriate substitutions made (**N.B.** not necessary if built through CMake):
+
+* Linux
+  
+  From within the built HTML directory (`documentation/user_docs/build/html`) run:
+     
+  `sed -r -i '/\[NULL\]/d' *html`
```

Comment:
do you need backticks?

Reply:
Those backticks put it in a monospaced codeblock in the rendered markdown